core/src/main/java/software/amazon/awssdk/core/internal/http/pipeline/stages/MakeRequestImmutable.java

```diff
@@ -15,10 +15,12 @@
 
 package software.amazon.awssdk.core.internal.http.pipeline.stages;
 
+import software.amazon.awssdk.annotations.SdkInternalApi;
 import software.amazon.awssdk.core.internal.http.RequestExecutionContext;
 import software.amazon.awssdk.core.internal.http.pipeline.RequestPipeline;
 import software.amazon.awssdk.http.SdkHttpFullRequest;
 
+@SdkInternalApi
 public class MakeRequestImmutable implements RequestPipeline<SdkHttpFullRequest.Builder, SdkHttpFullRequest> {
```

Comment:
Minor but these don't have Stage suffix but the others do.

Reply:
Fixed